client/gutenberg/extensions/contact-info/edit.js

```diff
@@ -0,0 +1,63 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { Component } from '@wordpress/element';
+import { InnerBlocks } from '@wordpress/editor';
+
+/**
+ * Internal dependencies
+ */
+const ALLOWED_BLOCKS = [
+	'jetpack/markdown',
+	'jetpack/address',
+	'jetpack/email',
+	'jetpack/phone',
+	'jetpack/map',
```

Comment:
Shall we add some other Jetpack blocks here, too?

Perhaps this list could be shared with the Forms block?

Reply:
I tried it and it didn't work very well. :/